README.md

```diff
@@ -58,5 +58,7 @@ Place for collecting great projects/capstones from previous springboard students
 | [Blog](https://learn.springboard.com/school-of-data/whitepapers/using-data-analysis-uncover-biases-funding-medical-research/) | "Using Data Analysis to Uncover Biases in Funding for Medical Research" |
 | [GitHub](https://github.com/jayguptacal/RetailAndMarketing/blob/main/HouseSalesPricePrediction/readme.md) | "Development of Real Estate House Prediction Demo App from the Proof of Concept (PoC) to Deployment Stage" By [jayguptacal](https://github.com/jayguptacal) |
 | [Blog](https://learn.springboard.com/using-netflix-inspired-algorithm-create-recommendation-engine-for-rock-climbers/) | "Using A Netflix-Inspired Algorithm To Create A Recommendation Engine For Rock Climbers" |
-| [Blog](https://learn.springboard.com/using-machine-learning-model-bring-black-white-tv-shows-to-life/) | "Using a Machine Learning Model to Bring Black-and-White TV Shows to Life" |
```

Comment:
@stephanied-springboard why is this one being deleted?

Reply:
Hi Andrew! I had mentioned this to you a few months ago, but this capstone is specifically for a student who is part of our University Partnerships program. We needed to remove it from our B2C site, so the link should now be down